From 07da345ac7f3d56d91bb786368dafe78d65ff618 Mon Sep 17 00:00:00 2001
From: John Stultz <john.stultz@linaro.org>
Date: Tue, 9 Apr 2013 10:30:26 -0700
Subject: [PATCH 265/634] pstore: Update Documentation/android.txt

Update Documentation/android.txt to reference PSTORE_CONSOLE
and PSTORE_RAM instead of ANDROID_RAM_CONSOLE

Change-Id: I2c56e73f8c65c3ddbe6ddbf1faadfacb42a09575
Reported-by: Jon Medhurst (Tixy) <tixy@linaro.org>
Signed-off-by: John Stultz <john.stultz@linaro.org>
---
 Documentation/android.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/android.txt b/Documentation/android.txt
index 72a62af..0f40a78 100644
--- a/Documentation/android.txt
+++ b/Documentation/android.txt
@@ -92,8 +92,8 @@ DNOTIFY
 1.3 Recommended enabled config options
 ------------------------------
 ANDROID_PMEM
-ANDROID_RAM_CONSOLE
-ANDROID_RAM_CONSOLE_ERROR_CORRECTION
+PSTORE_CONSOLE
+PSTORE_RAM
 SCHEDSTATS
 DEBUG_PREEMPT
 DEBUG_MUTEXES
-- 
1.9.1

